src/components/Card/README.md

```diff
@@ -0,0 +1,41 @@
+## Card
+An implementation of an Card for TPAs,
+The TPA implementation provides few extendable theme properties with the ability to override the base properties as well,
+
+By default the Card will pull the theme from the site (site colors),
+
+## Theme properties (All are optional)
+
+| propName   | propType | defaultValue | description |
+|------------|----------|--------------|-------------|
+| MainInfoColor  | string   | none | The color of the info container instead of the default color |
+| MainMediaColor  | string   | none | The color of the image container instead of the default color |
+| MainBorderColor  | string   | none | The color of the border instead of the default color |
+| MainBorderWidth  | string   | none | The width of the border instead of the default width |
+
+In order to override the theme, use Wix Stylable extends capabilities and wrap with stylable HOC:
+
+1. create an stylable file (e.g. DividerExt.st.css)
+    ` css
+    :import {
+        -st-from: "wix-ui-tpa/Card/Card.st.css";
+        -st-default: TPACard;
+    }
+
+    .root {
+        -st-mixin: TPACard(
+            MainInfoColor '"--infoColor"',
+            MainMediaColor '"--imageColor"',
+            MainBorderColor '"--borderColor"'
+            MainBorderWidth '"--borderWidth"'
+      );
+    }
+
+    `
+
+2. Create a component that uses it
+    ` javascript
```

Comment:
it should be a triple backtick <code>```</code>

Reply:
yeah, i'm trying to convert to mdx for the section api with no success so far